addons/auto-bumper/addon.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Auto-Bumper",
```

With this name, it sounds like topics could be bumped without user interaction. A more fitting name might be:
```suggestion
  "name": "One-click bump",
```